tests/pytests/test_isis_formatter.py

```diff
@@ -74,24 +74,30 @@ def sensor_position(self):
 
     @property
     def frame_chain(self):
-        j2000 = FrameNode(1)
+        frame_chain = FrameChain()
+
         body_rotation = TimeDependentRotation(
             np.array([[0, 0, 0, 1], [0, 0, 0, 1]]),
             np.array([0, 1]),
             100,
             1
         )
-        body_fixed = FrameNode(100, parent=j2000, rotation=body_rotation)
+        frame_chain.add_edge(100, 1, rotation=body_rotation)
+        frame_chain.add_edge(1, 100, rotation=body_rotation.inverse())
```

Comment:
It would be nice if we could tuck needing to add both edges behind a convenience function, or overloading add_edge.